doc/modules/cross_validation.rst

```diff
@@ -467,44 +376,189 @@ Here is a usage example::
 validation that allows a finer control on the number of iterations and
 the proportion of samples on each side of the train / test split.
 
+Cross-validation iterators with stratification based on class labels.
+=====================================================================
+
+For classification tasks, in some datasets the number of samples per class
+varies based on the class label. They are referred to commonly as datasets
+with imbalanced classes. In such cases, stratified sampling may be done to
+split the data in cross-validation.
+
+The :class:`StratifiedKFold` and :class:`StratifiedShuffleSplit` do
+stratified cross-validation, that is to ensure that each training and test set
+have the same class frequencies as the overall data (up to rounding).
+class is same as far as possible.
```

Comment:
Let's drop this paragraph. https://github.com/scikit-learn/scikit-learn/pull/6660/files#r78212466 is enough.